editor/themes/editor_theme_manager.cpp

```diff
@@ -282,27 +314,35 @@ EditorThemeManager::ThemeConfiguration EditorThemeManager::_create_theme_config(
 		if (config.preset != "Custom") {
 			Color preset_accent_color;
 			Color preset_base_color;
-			float preset_contrast = 0;
+			float preset_contrast = config.default_contrast;
 			bool preset_draw_extra_borders = false;
+			float preset_icon_saturation = config.default_icon_saturation;
 
-			// Please use alphabetical order if you're adding a new theme here.
-			if (config.preset == "Breeze Dark") {
+			// Please use alphabetical order if you're adding a new color preset here.
+			if (config.preset == "Black (OLED)") {
+				preset_accent_color = Color(0.45, 0.75, 1.0);
+				preset_base_color = Color(0, 0, 0);
+				// The contrast rate value is irrelevant on a fully black theme.
+				preset_contrast = 0.0;
+				preset_draw_extra_borders = true;
+			} else if (config.preset == "Blue") {
+				preset_accent_color = Color(0.44, 0.73, 0.98);
+				preset_base_color = Color(0.21, 0.24, 0.29);
+				preset_icon_saturation = 1;
+			} else if (config.preset == "Breeze Dark") {
 				preset_accent_color = Color(0.239, 0.682, 0.914);
 				preset_base_color = Color(0.1255, 0.1373, 0.149);
-				preset_contrast = config.default_contrast;
 			} else if (config.preset == "Godot 2") {
 				preset_accent_color = Color(0.53, 0.67, 0.89);
 				preset_base_color = Color(0.24, 0.23, 0.27);
```

Comment:
Considering this is a legacy classic theme, I believe a `preset_icon_saturation` set to 1 should be here too.